pytorch_lightning/loggers/mlflow.py

```diff
@@ -30,6 +30,7 @@
 try:
     import mlflow
     from mlflow.tracking import MlflowClient
+    from mlflow.tracking.context.registry import resolve_tags
```

Comment:
seems this is not supported in older versions

Reply:
thanks! my bad haven't thought of it upfront. trying to fix it...